server/load_store.go

```diff
@@ -52,12 +64,103 @@ func populateTargets(targets []string, s3 common.PrecomputedKeyStore, redis *red
 	return stores
 }
 
+func loadEigenDAV2Store(ctx context.Context, cfg CLIConfig, log logging.Logger) (*eigendav2.Store, error) {
+	// TODO: Replace with real logger once dependency PRs are merged
+
+	gethCfg := geth.EthClientConfig{
+		RPCURLs: []string{cfg.EigenDAConfig.EdaV1ClientConfig.EthRpcUrl},
+	}
+
+	ethClient, err := geth.NewClient(gethCfg, geth_common.Address{0x0}, 0, log)
+	if err != nil {
+		return nil, err
+	}
+
+	g1Points, err := kzg.ReadG1Points(cfg.EigenDAConfig.VerifierConfig.KzgConfig.G1Path, cfg.EigenDAConfig.VerifierConfig.KzgConfig.SRSNumberToLoad, 4)
+	if err != nil {
+		return nil, err
+	}
+
+	reader, err := eigenda_eth.NewReader(log, ethClient,
+		"0x0", cfg.EigenDAConfig.EdaV1ClientConfig.SvcManagerAddr)
+
+	if err != nil {
+		return nil, err
+	}
+
+	log.Info("Reading relays")
+	relayURLs, err := reader.GetRelayURLs(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	log.Info(fmt.Sprintf("Read relays %+v", relayURLs))
+
+	relayCfg := clients_v2.RelayClientConfig{
+		UseSecureGrpcFlag:  true,
+		MaxGRPCMessageSize: 10000000000,
+		Sockets:            relayURLs,
+	}
+
+	retriever, err := clients_v2.BuildRelayPayloadRetriever(log,
+		cfg.EigenDAConfig.V2RetrievalConfig,
+		gethCfg,
+		&relayCfg,
+		g1Points,
+	)
+
+	if err != nil {
+		return nil, err
+	}
+
+	// TODO: Sanitize properly
+	splits := strings.Split(cfg.EigenDAConfig.EdaV1ClientConfig.RPC, ":")
+
+	cfg.EigenDAConfig.V2DispersalConfig.SignerPaymentKey = cfg.EigenDAConfig.EdaV1ClientConfig.SignerPrivateKeyHex
+
+	cfg.EigenDAConfig.V2DispersalConfig.BlobCertifiedTimeout = time.Second * 100
+	cfg.EigenDAConfig.V2DispersalConfig.Quorums = []uint8{0, 1, 2}
```

Comment:
Why is `2` in this list?

Reply:
for some reason the old cert verifier we had deployed would only pass verification when 2 was provided as well. smth to do with a deployment error - would be nice to have a script that we can use for deploying our own 🥹